framework/components/ecl-site-headers/ecl-site-headers.twig

```diff
@@ -38,23 +42,23 @@
 {# Print result #}
 <header class="{{ _css_class }}" {{ _extra_attributes }} role="banner">
 
-  {% if links is defined %}
+  {% if user_menu.links is defined %}
   <div class="ecl-container">
     <!-- TODO: use utilities for float -->
     {% include '@ec-europa/ecl-navigation-lists' with {
-      title: title,
+      title: user_menu.title,
       variant: 'small',
-      links: links,
+      links: user_menu.links,
       extra_attributes: [{ name: 'style', value: 'float: right' }]
     } %}
   </div>
   {% endif %}
 
-  {% if political is defined and info is defined %}
+  {% if site_switcher.political is defined and site_switcher.info is defined %}
   {% include '@ec-europa/ecl-site-switchers' with {
```

Comment:
Please use the shortcut: `{% include '@ec-europa/ecl-site-switchers' with { variant: 'header' }|merge(site_switcher ) %}`